src/main/java/org/broadinstitute/barclay/utils/Utils.java

```diff
@@ -136,4 +141,31 @@ public static boolean equals(Object lhs, Object rhs) {
         return lhs == null && rhs == null || lhs != null && lhs.equals(rhs);
     }
 
+    /**
+     * a utility method to wrap multiple lines of text, while respecting the original
+     * newlines.
+     *
+     * @param input the String of text to wrap
+     * @param width the width in characters into which to wrap the text
+     * @return the original string with newlines added, and starting spaces trimmed
+     * so that the resulting lines fit inside a column of with characters.
+     */
+    public static String wrapParagraph(final String input, final int width)  {
+        final StringBuilder out = new StringBuilder();
+        String line;
+        try(final BufferedReader bufReader = new BufferedReader(new StringReader(input))) {
+            while ((line = bufReader.readLine()) != null) {
+                out.append(WordUtils.wrap(line, width));
+                out.append("\n");
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Because this should be unreachable, can you add a message for improving readability? Something like `throw new RuntimeException("Unreacheable statement", e); `.

Reply:
done.